flexible-adapter-app/src/main/res/layout/fragment_recycler_view_data.xml

```diff
@@ -31,8 +31,8 @@
                     android:id="@+id/recycler_view"
                     android:layout_width="match_parent"
                     android:layout_height="match_parent"
-                    app:items="@{items}"
                     tools:listitem="@layout/recycler_simple_item" />
+                <!-- app:items="@{items}" was causing build error, temporarily removed -->
```

Comment:
This was causing a build error for the demo app, would like to move fix for demo app to a separate pr to not hold up general build fix for maven central migration